src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Properties/InterceptedProjectProperties/ApplicationPropertyPage/ApplicationFrameworkValueProvider.cs

```diff
@@ -318,6 +318,27 @@ private async Task<string> GetPropertyValueAsync(string propertyName)
                     _ => unevaluatedPropertyValue
                 };
             }
+            else if (propertyName == MinimumSplashScreenDisplayTime)
+            {
+                if (int.TryParse(unevaluatedPropertyValue, out int value))
```

Comment:
Below, the code uses `Convert.ToInt16(unevaluatedPropertyValue)`, so I think you should be parsing as a `short` (Int16) here rather than an `int` (Int32).

```suggestion
                if (short.TryParse(unevaluatedPropertyValue, out short value))
```